documentation/assemblies/security/assembly-security.adoc

```diff
@@ -5,17 +5,22 @@
 [id='security-{context}']
 = Managing TLS certificates
 
-Strimzi supports encrypted communication between the Kafka and Strimzi components using the TLS protocol.
+[role="_abstract"]
+Strimzi supports TLS for encrypted communication between Kafka and Strimzi components.
+
 Communication between Kafka brokers (interbroker communication), between ZooKeeper nodes (internodal communication), and between those components and the Strimzi operators is always encrypted.
 Communication between Kafka clients and Kafka brokers is encrypted according to how the cluster is configured.
 For the Kafka and Strimzi components, TLS certificates are also used for authentication.
 
 The Cluster Operator automatically sets up and renews TLS certificates to enable encryption and authentication within your cluster.
 It also sets up other TLS certificates if you want to enable encryption or TLS authentication between Kafka brokers and clients.
-Certificates provided by users are not renewed.
 
-You can provide your own server certificates, called _Kafka listener certificates_, for TLS listeners or external listeners which have TLS encryption enabled.
-For more information, see xref:kafka-listener-certificates-str[].
+If you don't want to use the certificates generated by the Cluster Operator, you can xref:installing-your-own-ca-certificates-str[install your own cluster and client certificates].
```

Comment:
This is not completely correct. It is just the CA which the user provides. The CO will still generate the certificates. So it should be something like `If you don't want to use the CAs generated by the Cluster Operator, you can xref:installing-your-own-ca-certificates-str[install your own cluster and client CAs].`.

Reply:
I've updated as follows:

Certificate Authority (CA) certificates are generated by the Cluster Operator to verify the identities of components and clients. If you don't want to use the CAs generated by the Cluster Operator, you can xref:installing-your-own-ca-certificates-str[install your own cluster and client CA certificates].